src/main/scala/tech/cryptonomic/conseil/util/EffectsUtil.scala

```diff
@@ -0,0 +1,26 @@
+package tech.cryptonomic.conseil.util
+
+import cats._
+import cats.arrow.FunctionK
+import cats.effect.IO
+import scala.concurrent.Future
+import slick.dbio.DBIO
+import scala.concurrent.ExecutionContext
```

Comment:
I believe this ExecutionContext is not used here

Reply:
Will check thanks